docker-compose.prod-dev.yml

```
@@ -0,0 +1,12 @@
+# Builds on docker-compose.yml and docker-compose.dev.yml
+# Runs the prod image in like dev mode, but as if it was deployed to staging.
+
+services:
+  civiform:
+    build: prod
+    image: civiform/civiform
```

```suggestion
    image: civiform
```

You should use the local `civiform` name. It's consistent with the other compose files and won't risk accidentally pulling a new latest when you don't want it.